src/libstore/repair-flag.hh

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+namespace nix {
+
+enum RepairFlag : bool { NoRepair = false, Repair = true };
```

Comment:
This is sort of silly, but is the one definition used by `store-api.hh` and `derivations.hh`. I cannot make it abstract because the `NoRepair` default arguments.